launch/docker.go

```diff
@@ -36,25 +38,23 @@ func newDocker(setupImage, setupImageVer string) runner {
 }
 
 func (d *docker) setupBin() error {
-	_ = execCommand("docker", "volume", "rm", "--force", d.volume).Run()
+	_ = execDockerCommand("volume", "rm", "--force", d.volume)
 
-	err := execCommand("docker", "volume", "create", "--name", d.volume).Run()
+	err := execDockerCommand("volume", "create", "--name", d.volume)
 	if err != nil {
-		return fmt.Errorf("failed to create docker volume")
+		return fmt.Errorf("failed to create docker volume: %v", err)
 	}
 
 	mount := fmt.Sprintf("%s:/opt/sd/", d.volume)
 	image := fmt.Sprintf("%s:%s", d.setupImage, d.setupImageVersion)
-	err = execCommand("docker", "pull", image).Run()
+	err = execDockerCommand("pull", image)
```

Comment:
[note] it looks stuck for a long time while pulling image.

Reply:
Is it better to output log about pulling image?
Can I merge this PR as it is?

Reply:
We can do it in another PR so it's fine to merge :+1: